src/logic/validate.ts

```diff
@@ -1,21 +1,66 @@
-import { BUNDLED_THEMES } from 'shiki';
-import type { ValidateFuncOptions } from '../types/function';
+import { BUNDLED_LANGUAGES, BUNDLED_THEMES } from 'shiki';
+import type { ValidOptions } from '../types/function';
+
+const validateBoolean = (name: string, bool: boolean, required: boolean): string => {
+  if (!bool && required) return `\`${name}\` is required!`;
+  if (bool && typeof bool !== 'boolean') return `${name} must be a boolean!`;
+  return '';
+};
```

Comment:
dic, can you test this one out? line 6 kalo `bool` value nya false, kan dia bakal return error ya.

Reply:
oiya